sources/osg/Matrix.js

```diff
@@ -604,6 +640,24 @@ var Matrix = {
         };
     } )(),
 
+    preMultRotate: function ( matrix, quat ) {
+        if ( Quat.zeroRotation( quat ) )
```

Comment:
I'm not sure I get you. 

Do you want me to  

1. Subsitute `Quat.zeroRotation( quat )`  by   ` quat[ 0 ] === 0.0 && quat[ 1 ] === 0.0 && quat[ 2 ] === 0.0 && quat[ 3 ] === 1.0 `  
2. Put r matrix in a closure. 
3. Both.